lucille-core/src/test/java/com/kmwllc/lucille/core/HeartbeatTest.java

```diff
@@ -48,6 +57,18 @@ public void testWatcher() throws Exception {
 
     String lastLine = lines.skip(currentLineCount - 1).findFirst().get();
     assertTrue(lastLine.contains("INFO Heartbeat: Issuing heartbeat"));
+
+    //Thread.sleep(1000);
+
+    Collection<Thread> nonSystemThreadsAfter =
+        ThreadUtils.findThreads(t -> !ThreadUtils.getSystemThreadGroup().equals(t.getThreadGroup()));
+
+    for (Thread thread : nonSystemThreadsAfter) {
+      log.info("THREAD testWatcher AFTER: {}", thread.getName());
+      if (thread.getName().startsWith(ThreadNameUtils.THREAD_NAME_PREFIX)) {
```

Comment:
suggest adding a boolean method to ThreadNameUtils called isLucilleThread() and using it here
it can do the test in the same way

Reply:
resolved in [2044c9f](https://github.com/kmwtechnology/lucille/pull/172/commits/2044c9f47eab57317e613780caed80c0de5958fd)